src/daft-plan/src/builder.rs

```diff
@@ -0,0 +1,40 @@
+use std::sync::Arc;
+
+use crate::logical_plan::LogicalPlan;
+use crate::{ops, source_info};
+
+#[cfg(feature = "python")]
+use daft_core::python::schema::PySchema;
+#[cfg(feature = "python")]
+use pyo3::prelude::*;
```

Comment:
Also, if the number of these `use`s increases, we can have a single attribute specification apply to an entire block of imports using this syntax: https://github.com/Eventual-Inc/Daft/blob/d032844dd68c694a4f36eb33b780e444dae76dae/src/daft-core/src/array/ops/cast.rs#L27-L40

Reply:
Actually, I think you were the one to make that change in your `Duration` dtype PR, I didn't know about that syntax until I saw your change! 😄